src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java

```diff
@@ -33,11 +33,9 @@
 
 	private JButton Play, Instruction, Exit, Resume, Easy, Medium, Hard, Back, Menu, Credit, PlayBGM, PauseBGM, HighScore, NextPage;
 	private JButton Character, CMenu, Kwhale, Shark;
-	private JFrame frame, instruct;
+	public static JFrame frame, instruct;
```

Comment:
What happened here?  They used to be private members, and now they're public statics?  This is almost definitely wrong.  You'll want to revert this.

Reply:
it is becasue it was used in my other pull request that I correct the glitchy frames. I have to access the frame from other classes.

Reply:
fixed